src/utils/request_cache_drop.py

```diff
@@ -0,0 +1,64 @@
+# this module accepts environment variables which control
+# cache dropping behavior
+# cache dropping is implemented by privileged pods separate from
+# the benchmark pods, and these are invoked by 
+# HTTP GET from here.
+# if no environment variables are defined, nothing happens
+
+import os
+import http.client
+import sys
+import logging
+
+class RunSnafuCacheDropException(Exception):
+    pass
+
+dropKernelCachePort = 9435  # FIXME: hardcoded for now
+dropCephCachePort = 9437  # FIXME: hardcoded for now
+logger = logging.getLogger("snafu")
+
+dbgLevel = os.getenv('DROP_CACHE_DEBUG_LEVEL') 
+if dbgLevel != None:
+    logger.setLevel(logging.DEBUG)
+    logger.info('drop_cache debug log level')
+
+http_debug_level = int(os.getenv('HTTP_DEBUG_LEVEL', default=0))
+
+http_timeout = 10
+
+def drop_cache():
+
+    # drop Ceph OSD cache if requested to
+    
+    ceph_cache_drop_pod_ip = os.getenv('ceph_drop_pod_ip')
```

Comment:
This env var should be `ceph_cache_drop_pod_ip` as stated in https://github.com/cloud-bulldozer/snafu/blob/master/src/fio_wrapper/fio_wrapper.py#L55

Reply:
Now that ceph cache drop is implemented in request_cache_drop.py I think we can get rid of the old code:
https://github.com/cloud-bulldozer/snafu/blob/master/src/fio_wrapper/fio_wrapper.py#L71-L81

What do you think?

Reply:
correct, was going to do that in the next PR that was going to do the same thing to ceph cache dropping - pull it out of individual benchmark and make it common code usable by any benchmark that wants it.

Reply:
Is there any way to pull out the steps in roles/fio_distributed/tasks/main.yml that start the kernel cache drop pods and capture the kernel cache drop pod IPs and make them a "subroutine" that gets called by the benchmark instead?  Would that be a role in ansible?